apps/service_providers/llm_service/state.py

```diff
@@ -100,6 +108,20 @@ def get_prompt(self):
     def get_tools(self):
         return get_tools(self.session, self.experiment)
 
+    def get_trace_metadata(self) -> dict:
+        if trace_service := self.experiment.get_trace_service():
+            trace_info = trace_service.get_current_trace_info()
```

Comment:
Does this return info about the trace it's busy with, or just general info?

Reply:
the one it's busy with